.github/workflows/contracts-deploy.yml

```diff
@@ -55,7 +64,7 @@ jobs:
       (github.event_name == 'pull_request' &&
        github.event.action == 'labeled' &&
        github.event.label.name == '[AZERO] DEPLOY-CONTRACTS')
-    runs-on: ubuntu-20.04
+    runs-on: self-hosted
```

Comment:
If we switch to self-hosted runner, we can install all those pre-requisites (jq, node etc) upfront so we save overall time. 
CC @mikogs

Reply:
- jq definitely
- whatever LTS version of node is the most current

Reply:
@Marcin-Radecki Both node16 and jq are there on the AWS runners, along with many others.  It should be just fine.

Reply:
Right, thanks